### DISCUSSION TITLE
SQL文件大小

### DISCUSSION THREAD

**Discussion by wujuwu** [hhyo/Archery#2074] [Q&A] [answered]
求问个问题：我的版本是1.9.1，我修改了sql/templates/sqlsubmit.html内限制上传文件的大小为50M，但是SQL检测也不能通过，请问最大SQL文件是多少？要设置多少合适？

**Comment by nick2wang** [comment, marked as answer]
还是不建议上传大文件sql，检测时会占用较多的系统资源
https://github.com/hhyo/Archery/blob/b18de11fb3d5295f92c32eedd5f4b2c2e78fb72a/archery/settings.py#L46-L47

**Reply by wujuwu** [reply]
求问15728640是多少M？我想试下检测最多多少合适

**Reply by nick2wang** [reply]
> 求问15728640是多少M？我想试下检测最多多少合适

15728640/1024/1024=15M

**Reply by wujuwu** [reply]
好的，还能设置大点吗，比如40M的话会不会能通过？

**Reply by nick2wang** [reply]
可以，不要超过archery mysql的max_allowed_packet

**Reply by wujuwu** [reply]
我在配置文件里看到默认的max_allowed_packet=33554432  是32M吗，我最大只能配置32M，还是说我可以把max_allowed_packet改成50M？然后把DATA_UPLOAD_MAX_MEMORY_SIZE也改成50M？
